app/views/handlers/list_collector.py

```diff
@@ -54,6 +54,7 @@ def handle_post(self):
             # pylint: disable=no-member
             # wtforms Form parents are not discoverable in the 2.3.3 implementation
             self.questionnaire_store_updater.update_answers(self.form.data)
+            self.questionnaire_store_updater.update_progress_for_dependent_sections()
```

Comment:
Is this needed?. As before, length won't change on a collector page?

Reply:
Not needed, thanks.